### PR TITLE
Google Login Fix

### PR DESCRIPTION
**Pull request by JoshuaVSherman**

### CODE REVIEW

File: env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+PORT=9000
+NODE_ENV=development
```
Comment (JoshuaVSherman) [inline env]: @m0cah never check in your env variables to github, please remove this file

File: package.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
     "@emotion/styled": "^11.8.1",
     "@mui/material": "^5.4.2",
     "@popperjs/core": "^2.9.2",
+    "@react-oauth/google": "^0.2.6",
```
Comment (JoshuaVSherman) [inline package.json:45]: @m0cah This file still says Change In Christ, please update
and you have not removed the old react-google-login package either, please do

File: src/App/AppTemplate/MenuItem.tsx [hunk 3]
```diff
@@ -2,67 +2,54 @@ import React from 'react';
...
+import { Button } from '@mui/material';
 
 interface IgoogleButtonsProps {
   type: string, index: string | number | undefined,
-  auth:any, dispatch:(arg0:any)=>void,
+  auth: any, dispatch: (arg0: any) => void,
 }
-function GoogleButtons(props:IgoogleButtonsProps): JSX.Element {
+
+const login = useGoogleLogin({
```
Comment (JoshuaVSherman) [inline src/App/AppTemplate/MenuItem.tsx]: @m0cah the hook here needs some parameters that are missing, you need to call it inside of the GoogleButtons function so you can pass in those, I will make a commit to fix this

**Review by m0cah — COMMENTED** [review]
Good morning Josh, I self-reviewed this and it includes everything but unit tests. I will create a future ticket for that.

**Comment by JoshuaVSherman** [comment]
@m0cah it does not look like you have completed the requested changes from my merge request, please let me know if you need help, please do those changes asap, before moving on to a new issue, thank you

File: env [hunk 4]
```diff
@@ -0,0 +1,9 @@
+PORT=9000
```
Comment (JoshuaVSherman) [inline env]: this env should not be committed to github, please remove

File: src/App/AppTemplate/MenuItem.tsx (hunk 3 above)
Comment (m0cah) [reply src/App/AppTemplate/MenuItem.tsx]: This change is fixed I believe.

File: package.json (hunk 2 above)
Comment (m0cah) [reply package.json:45]: The old package was not there for me. I had removed it awhile ago. I searched for it using ctrl f and it wasn't found.

File: env (hunk 1 above)
Comment (m0cah) [reply env]: Fixed in latest commit.

File: env (hunk 4 above)
Comment (m0cah) [reply env]: Fixed in latest commit.

**Pull request merged**